### ISSUE TITLE
Gamepad view in main window

### ISSUE THREAD

**Issue by NyakoFox** [cemu-project/Cemu#652]
It'd be cool if the gamepad view could be in the main window, like what Citra can do with the 3ds' multiple screens. This would make it a lot easier to play games like Paper Mario: Color Splash, where both are needed often, while screensharing on Discord, since you can't screenshare two windows and still have audio.

**Comment by NyakoFox** [comment]
Yes, I meant both at the same time. I know about both of those, but I found it a bit awkward to use because I need to look at both screens at once, but can't use the different window thing because I still want to screenshare the game.

**Comment by cbartondock** [comment]
One should also mention that CTRL+TAB to toggle seems to be broken at present.
